### DISCUSSION TITLE
How to close remote host?

### DISCUSSION THREAD

**Discussion by icarusion** [coder/code-server#5968] [Q&A]
When I update to 4.9.1(Code: 1.73.1)，the remote host is opening as default, I want to know how to close it?
<img width="373" alt="image" src="https://user-images.githubusercontent.com/5370542/212220791-bed1e555-4759-4047-8cd0-79aa44f6abfe.png">

**Comment by lg-nineoneninethreenine** [comment]
That is expected behavior afaik, it happens on my code-server too.


But wait, @jsjoeio is it expected behavior? It is... Right?

**Reply by jsjoeio** [reply]
I think so? cc @code-asher

**Reply by code-asher** [reply]
Yeah seems expected to me.  code-server is itself remote which is why that shows up I believe.  Closing it would probably end up closing the browser window.

What is the desired outcome in being able to close the remote?  Is it mostly to gain a bit more space/reduce visual clutter?

**Reply by code-asher** [reply]
Ah actually I think in an older version there was no remote button but upstream must have added one?  Actually I know there used to be one so it is possible they removed it then added it back.  But since code-server is indeed a remote host it might make sense to leave it.

**Reply by lg-nineoneninethreenine** [reply]
Maybe we can remove the button from the status bar?

**Reply by code-asher** [reply]
Yeah that would make sense.  I think this is something we would want to suggest upstream though; to me it feels like the justification is not strong enough to warrant a patch.
